taipy/core/notification/_core_event_consumer.py

```diff
@@ -16,7 +16,7 @@
 from .event import Event
 
 
-class CoreEventConsumerBase(threading.Thread):
+class _CoreEventConsumerBase(threading.Thread):
```

Comment:
Since this is now a hidden class, should we update the docstring and move it somewhere else?

Reply:
Since the class name starts with an underscore, it is not used in taipy-doc. There is no need to remove it.